app/src/main/res/layout/activity_market_picker.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:id="@+id/topBar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/background"
+        android:elevation="@dimen/top_bar_elevation"
+        android:gravity="center_horizontal"
+        android:padding="@dimen/base_margin">
+
+        <ImageView
+            android:id="@+id/close"
```

Comment:
Should this screen have a close-button? Given that it's mandatory to fill out the data requested in it 🙂

Reply:
its not, I just forgot to remove the ID when i copied some stuff :P